src/miner.cpp

```diff
@@ -490,7 +490,7 @@ void BlockAssembler::addPackageTxs(int &nPackagesSelected, int &nDescendantsUpda
             packageSigOpsCost = modit->nSigOpCostWithAncestors;
         }
 
-        if (packageFees < blockMinFeeRate.GetFee(packageSize)) {
+        if (packageFees < blockMinFeeRate.GetRelayFee(packageSize)) {
```

Comment:
Since this changes, I think we should also change `src/test/miner_tests.cpp` on line 124 and 161